tests/test_solax.py

```diff
@@ -1,6 +1,8 @@
-from unittest.mock import Mock
 import asyncio
```

Comment:
It seems like there are no functional changes here, just reordering the imports. As mentioned above, this pollutes the diff.